apiary.apib

```diff
@@ -3559,6 +3559,64 @@ Update the current timer. Only possible if there is a timer running.
 
 ## Work Orders [/workOrders]
 
+### workOrders.list [GET /workOrders.list]
+
+Get a list of work orders.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + starts_at
+                + order (enum[string], optional)
+                    + Members
+                        + asc
+                        + desc
+            + Default
+                + (object)
+                    + field: `starts_at`
+                    + order: `asc`
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string)
+                + state (enum[string])
+                    + Members
+                        + draft
+                        + finalized
+                        + sent
+                + customer (object)
+                    + type (enum[string])
+                        + Members
+                            + contact
+                            + company
+                    + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string)
+                + starts_at: `2019-05-08T10:12:01+00:00` (string)
+                + description: `Pipework intervention` (string)
```

Comment:
B: needs to be nullable (see .info). Repeated for department, milestone, event, location and distance.